File: app/src/main/java/amrabed/android/release/evaluation/edit/Modification.java
package amrabed.android.release.evaluation.edit;

import amrabed.android.release.evaluation.data.entities.Task;

public class Modification {
    static final int ADD = 0;
    static final int UPDATE = 1;
    static final int DELETE = 2;

    public final Task task;
    final int operation;

    Modification(Task task, int operation) {
        this.task = task;
        this.operation = operation;
    }
}

File: app/src/main/java/amrabed/android/release/evaluation/edit/Modification.kt
package amrabed.android.release.evaluation.edit

import amrabed.android.release.evaluation.data.entities.Task

class Modification internal constructor(val task: Task?, val operation: Int) {

    companion object {
        const val ADD = 0
        const val UPDATE = 1
        const val DELETE = 2
    }

}